task7/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script src="js/script.js"></script>
+</body>
+</html>
```

Comment:
Please revert deleted tsk3 files, don't delete them, just move to common `UI` folder and use existing HTML file with all the product structure